.eslintrc.js

```diff
@@ -17,4 +17,16 @@ module.exports = {
     '@npmcli',
     ...localConfigs,
   ],
+  rules: {
+    'import/no-extraneous-dependencies': ['error', { devDependencies: false }],
+  },
+  overrides: [
+    {
+      files: ['**/test/**', '.eslintrc.js', '.eslintrc.local.js'],
```

Comment:
making an assumption this covers everything that's not considered package src. Not sure if that's actually the case?

Reply:
The standard that templateOSS sets up is

```json
  "files": [
    "bin/",
    "lib/"
  ],
```

It's much easier to explicitly specify where the code is than where it isn't.  The rule should apply to those two directories and the handful of projects that need to override this can do so.

Reply:
Ah that’s good to know. Shall I open a pr on the config repo then for these folders?

Reply:
Yeah let's start there.

Reply:
My glob is rusty but something like this is what I had in mind to start
``"import/no-extraneous-dependencies": ["error", {"devDependencies": ["!lib/**", "!bin/**"]}]``

Reply:
Opened https://github.com/npm/eslint-config/pull/67